webplugin/js/app/apz-client-1.0.js

```
@@ -592,35 +538,27 @@ window['APPLOZIC'] ||
-                            var messageFeed = apzMessageUtils.getMessageFeed(
-                                resp.message
-                            );
+                            var messageFeed = apzMessageUtils.getMessageFeed(resp.message);
                             events.onMessageReceived({
                                 message: messageFeed,
                             });
                         } else if (messageType === 'APPLOZIC_02') {
-                            var messageFeed = apzMessageUtils.getMessageFeed(
-                                resp.message
-                            );
+                            var messageFeed = apzMessageUtils.getMessageFeed(resp.message);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'messageFeed'. Consider to delete it or rename it.

'messageFeed' is defined here:


(lint/suspicious/noRedeclare)